pkg/codegen/schema/loader_server.go

```diff
@@ -52,7 +52,28 @@ func (m *loaderServer) GetSchema(ctx context.Context,
 		version = &v
 	}
 
-	pkg, err := m.loader.LoadPackage(req.Package, version)
+	descriptor := &PackageDescriptor{
```

Comment:
Surprised nothing needed this before. This is small enough and obvious enough we could merge this change as is if it was its own PR.

Reply:
It is in it's own pr now to be submit separately #18021